ApiDocs.Console/Program.cs

```diff
@@ -937,16 +956,48 @@ internal static void WriteValidationError(string indent, ValidationError error)
                 httpLogging.ClosePackage();
             }
 
-            // TODO: Print out account summary if multiple accounts were used
-            foreach(var key in results.Keys)
+            // Print out account summary if multiple accounts were used
+            foreach (var key in results.Keys)
             {
                 FancyConsole.Write("Account {0}: ", key);
                 results[key].PrintToConsole(false);
             }
 
+            // Print out undocumented properties, if any where found.
+            WriteUndocumentedProperties();
+
             return !results.Values.Any(x => x.WereFailures);
         }
 
+        private static void WriteUndocumentedProperties()
+        {
+
+            Dictionary<string, List<string>> undocumentedProperties = new Dictionary<string, List<string>>();
+            foreach (var props in DiscoveredUndocumentedProperties)
+            {
+                List<string> foundPropertiesOnResource;
+                if (!undocumentedProperties.TryGetValue(props.ResourceName, out foundPropertiesOnResource))
+                {
+                    foundPropertiesOnResource = new List<string>();
+                    undocumentedProperties.Add(props.ResourceName, foundPropertiesOnResource);
+                }
+                if (!foundPropertiesOnResource.Contains(props.PropertyName))
```

Comment:
make it a HashSet<string> instead of a List<string> and you don't need to worry about the `contains` check